src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/ManageOffencesService.kt

```diff
@@ -15,4 +16,8 @@ class ManageOffencesService(
   fun getSdsExclusionsForOffenceCodes(offenceCodes: List<String>): List<SDSEarlyReleaseExclusionForOffenceCode> {
     return manageOffencesApiClient.getSdsExclusionsForOffenceCodes(offenceCodes)
   }
+
+  fun getOffences(offenceCodes: List<String>): List<Offence> {
```

Comment:
Is it possible to change the offenceCodes param to be a Set rather than a List? - to avoid having to check for duplicates